proxy/proxy/salt-broker/salt-broker

```diff
@@ -175,11 +187,6 @@ class AbstractChannelProxy(multiprocessing.Process):
                     self.backend_ever_connected = True
                     self.reconnect_retries = self.opts["drop_after_retries"]
                 elif mon_evt["event"] == zmq.EVENT_CONNECT_RETRIED:
-                    if (
-                        not self.backend_ever_connected
```

Comment:
IIUC with this change then `self.backend_ever_connected` won't be really needed, isn't it?

Reply:
Yes, it's true, thanks for pointing :+1: